src/main/drivers/accgyro_spi_mpu6000.c

```diff
@@ -104,6 +104,9 @@ static bool mpuSpi6000InitDone = false;
 
 static IO_t mpuSpi6000CsPin = IO_NONE;
 
+#define SPI_CLOCK_MPU6000_INITIALIZATION SPI_CLOCK_INITIALIZATON
+#define SPI_CLOCK_MPU6000_FAST           SPI_CLOCK_FAST
```

Comment:
You may never see me creating another PR. Fixing bugs is fun. Making a PR is not. End of story.